default.nix

```diff
@@ -48,15 +40,26 @@ resholve.mkDerivation rec {
           # cannot be reached when built with nix
           "shasum"
         ];
+        external = [
+          # We want to reference the ambient Nix when possible, and have custom logic
+          # for the fallback
+          "nix"
+        ];
       };
       keep = {
         "$cmd" = true;
         "$direnv" = true;
+
+        # Nix fallback implementation
+        "$_nix_direnv_nix" = true;
+        "$ambient_nix" = true;
+        "$NIX_DIRENV_FALLBACK_NIX" = true;
       };
-      execer = [
-        "cannot:${direnv}/bin/direnv"
-        "cannot:${nix}/bin/nix"
-      ];
+      prologue =
+        (writeText "prologue.sh" ''
+          NIX_DIRENV_SKIP_VERSION_CHECK=1
+          NIX_DIRENV_FALLBACK_NIX=''${NIX_DIRENV_FALLBACK_NIX:-${lib.getExe nix}}
```

Comment:
Can you mention this environment variable in the README? Otherwise it looks good to me.

Reply:
Done.